adapter/config/default_config.go

```diff
@@ -35,7 +35,7 @@ var defaultConfig = &Config{
 		},
 		VhostMapping: []vhostMapping{
 			{
-				Environment: "Default",
+				Environment: "default",
```

Comment:
shall we remove VhostMapping as we get it now from gateway cr.

Reply:
removed and resolved